### PR TITLE
fix row_conv_op small size unittest cases

### PR DESCRIPTION
**Pull request by LDOUBLEV**
fix 2 cases in row_conv_op, enlarge  wt input size to satisfy shape >=100.

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/test_row_conv_op.py [hunk 1]
```diff
@@ -139,11 +139,11 @@ def row_conv_foward_Tensor(x, wt):
 class TestRowOpWithTensorInput(OpTest):
     def setUp(self):
         self.op_type = "row_conv"
-        length = [3, 2, 4]
-        B = 2
+        length = [2, 2, 2]
+        B = 1
```
Comment (zhupengyang) [inline python/paddle/fluid/tests/unittests/test_row_conv_op.py]: length间隔取不一样的好一点？batch取个大于1的值会更好？

**Review by zhupengyang — APPROVED** [review]
LGTM

File: python/paddle/fluid/tests/unittests/test_row_conv_op.py (hunk 1 above)
Comment (LDOUBLEV) [reply python/paddle/fluid/tests/unittests/test_row_conv_op.py]: Done

**Review by hong19860320 — APPROVED** [review]
LGTM

**Pull request merged**